test/packages/deploy_custom_agent_multi_services/data_stream/memory/_dev/deploy/agent/custom-agent.yml

```diff
@@ -0,0 +1,18 @@
+version: '2.3'
+services:
+  docker-custom-agent:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: elastic-agent-oracle-client-1
+    healthcheck:
+      test: ["CMD", "bash", "-c", "echo 'select 1 from dual;' | ORACLE_HOME=/opt/oracle/instantclient_21_4 /opt/oracle/instantclient_21_4/sqlplus -s sys/Oradoc_db1@oracle:1521/ORCLCDB.localdomain?sysdba=1"]
+      interval: 10s
+      timeout: 10s
+      retries: 60
+    user: root
+  oracle:
+    image: docker.elastic.co/observability-ci/database-enterprise:12.2.0.1
+    ports:
+      - 1521:1521
+      - 5500:5500
```

Comment:
These files are added into the data stream directory, shouldn't these files be added into the `_dev/deploy/agent` directory under the root of the package?

Reply:
We can have it added at two places. Under the datastream and like you mentioned at the _dev/deploy/agent

The second is a more advanced usecase that was needed in oracle and hence thought of putting that as an example here. 

Should i go ahead as it is or put files under `_dev/deploy/agent`. If latter approach, i may have to delete the custom-agent.yml file in `_dev/deploy/agent` which is unnecessary

Reply:
Oracle PR  : https://github.com/elastic/integrations/pull/4461

Reply:
Ah, you are right both locations lgtm.